README.md

```diff
@@ -73,6 +74,9 @@ At the conclusion, you will see a summary of your entries along with an option t
 If you see a mistake and have to repeat the dialogue, the default values are conveniently changed to your previous entries, so that you won't be forced to retype correct entries. When finished, Radiian will create an Ansible playbook that is ready for immediate deployment to AWS. The output will read:
 `Your playbook has been created in the directory ./ansible`
 
+### <a name="appropriate-permissions"></a> Setting Up An IAM Machine User
+You need to have an IAM machine user in order for the playbook to access your AWS account. That user must have various permissions in order for it to access your AWS account. When setting up your IAM machine user, you want to abide by the principle of least privilege. You can check out best practices [here](http://radify.io/blog/rise-of-the-machine-users/). [This](https://gist.github.com/gavD/c9f680c9058e48c20c122ee69e52844b#v3) is an example of an IAM Policy that you could use for your machine user that provides Ansible with enough power to run your immutable infrastructure without opening the front-door security wise.
```

Comment:
This example I gave is kinda global...

This is a better policy, more locked down, but make it clear that they have to specify their account ID and one or more subnet IDs:

```json
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "EC2ReadOnly",
            "Effect": "Allow",
            "Action": [
                "ec2:Describe*",
                "elasticloadbalancing:Describe*"
            ],
            "Resource": [
                "*"
            ]
        },
        {
            "Sid": "EC2ControlAll",
            "Effect": "Allow",
            "Action": [
                "ec2:CreateTags",
                "elasticloadbalancing:RegisterInstancesWithLoadBalancer"
            ],
            "Resource": [
                "*"
            ]
        },
        {
            "Sid": "EC2Control",
            "Effect": "Allow",
            "Action": [
                "ec2:RunInstances",
                "ec2:TerminateInstances"
            ],
            "Resource": [
                "arn:aws:ec2:us-east-1:<YOUR ACCOUNT ID>:instance/*",
                "arn:aws:ec2:us-east-1:<YOUR ACCOUNT ID>:subnet/<YOUR SUBNET ID HERE>",
                "arn:aws:ec2:us-east-1:<YOUR ACCOUNT ID>:volume/*",
                "arn:aws:ec2:us-east-1:<YOUR ACCOUNT ID>:network-interface/*",
                "arn:aws:ec2:us-east-1:<YOUR ACCOUNT ID>:key-pair/*",
                "arn:aws:ec2:us-east-1:<YOUR ACCOUNT ID>:security-group/*",
                "arn:aws:ec2:us-east-1::image/ami-*"
            ]
        }
    ]
}
```